elasticjob-cloud/elasticjob-cloud-scheduler/src/main/java/org/apache/shardingsphere/elasticjob/cloud/scheduler/mesos/FacadeService.java

```diff
@@ -136,17 +132,15 @@ public void addRunning(final TaskContext taskContext) {
     
     /**
      * Update daemon task status.
-     * 
      * @param taskContext task running context
-     * @param isIdle set to idle or not
+     * @param isIdle      set to idle or not
      */
     public void updateDaemonStatus(final TaskContext taskContext, final boolean isIdle) {
         runningService.updateIdle(taskContext, isIdle);
     }
     
     /**
      * Remove task from running queue.
-     *
```

Comment:
Please keep original java doc format, keep a blank line between title and parameter